wcfsetup/install/files/lib/acp/form/ReactionTypeEditForm.class.php

```diff
@@ -0,0 +1,34 @@
+<?php
+namespace wcf\acp\form;
+use wcf\data\reaction\type\ReactionType;
+use wcf\system\exception\IllegalLinkException;
+
+/**
+ * Represents the reaction type add form.
+ *
+ * @author	Joshua Ruesweg
+ * @copyright	2001-2018 WoltLab GmbH
+ * @license	GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @package	WoltLabSuite\Core\Acp\Form
+ * @since	3.2
+ */
+class ReactionTypeEditForm extends ReactionTypeAddForm {
+	/**
+	 * @inheritDoc
+	 */
+	public $formAction = 'update';
```

Comment:
This should be `edit`. The relevant comment in `AbstractFormBuilderForm` is incorrect.